test/functional/rpc_packages.py

```diff
@@ -377,6 +377,10 @@ def test_submitpackage(self):
         self.log.info("Submitpackage only allows packages of 1 child with its parents")
         # Chain of 3 transactions has too many generations
         legacy_pool = node.getrawmempool()
+
+        chain_hex_decode_failure = [t["hex"][:-1] + 'X' for t in self.wallet.create_self_transfer_chain(chain_length=2)]
```

Comment:
two empty string might be simpler?

Reply:
thanks! fair point updated in [4bdb78c](https://github.com/bitcoin/bitcoin/pull/31139/commits/4bdb78cfe76414227b749674ed660f4972fb6eda)